doc/community-build/Generate-Cortx-Build-Stack.md

```diff
@@ -17,10 +17,11 @@ To know about various CORTX components, see [CORTX Components guide](https://git
    cd /root && git clone https://github.com/Seagate/cortx --recursive --depth=1
    ```
 
-2. Run the following command to check out the codebase from the **main** branch for all components:
+2. Run one of the following commands to check out the codebase from the **main** branch for all components:
```

Comment:
Maybe we can add statement that uses docker images based on OS and provide tag details. Just to avoid `Run one of the following commands` multiple times. 
Also, the OS of the VM created and the tag has to be the same. centos-7.8.2003 image tag will not work with CentOS 7.9 VM.